source/index.html.md.erb

```diff
@@ -634,6 +636,7 @@ let bookings = api.bookings.get({
       "internal_id": "123-ABC",
       "creator_name": "Hollie B.",
       "creator_id": "69a66cb2-dcb2-4803-9d23-f7894e443e68",
+      "creator_profileimage": "api.snappr.com/rpc/v1/avatar/08c13d8d-005c-4a0c-a4cd-703816ed3960"
```

Comment:
are we sure this is the type of url we will be returning?, from what I see this method doesn't return an rpc endpoint ?
https://github.com/snappr/snappr.server/blob/dddb42245b0b7a8201861fbb370103ff401b2f9f/src/models/media/media.ts#L522

https://github.com/snappr/snappr.server/blob/dddb42245b0b7a8201861fbb370103ff401b2f9f/src/models/media/media.ts#L440

I know on the web app navbar this is the url we show, but this uses an old endpoint and method

Reply:
lets rename it to creator_profile_image to be consistend